src/v/config/throughput_control_group.cc

```diff
@@ -117,11 +117,14 @@ throughput_control_group::operator=(const throughput_control_group& other) {
 
 std::ostream&
 operator<<(std::ostream& os, const throughput_control_group& tcg) {
+    fmt::print(os, "{{");
+    if (!tcg.is_noname()) {
+        fmt::print(os, "group_name: {}, ", tcg.name);
+    }
     fmt::print(
       os,
-      "{{group_name: {}, client_id: {}, throughput_limit_node_in_bps: {}, "
+      "client_id: {}, throughput_limit_node_in_bps: {}, "
       "throughput_limit_node_out_bps: {}}}",
-      tcg.name,
```

Comment:
should we always write `group_name: ` but vary the value (e.g. `{}` vs `{group-name}`. having the group name label always present makes parsing log messages easier

Reply:
that makes sence, but is would be different from `{}` because it's different from an empty string

Reply:
`{{{}}}`?

Reply:
i mean it doesn't matter much how it is formatted